modules/perception/onboard/component/fusion_camera_detection_component.cc

```diff
@@ -1141,6 +1111,46 @@ int FusionCameraDetectionComponent::MakeCameraDebugMsg(
   return cyber::SUCC;
 }
 
+bool FusionCameraDetectionComponent::SetCameraHeight(
+     const std::string &sensor_name,
+     const std::string &params_dir, float default_camera_height,
+     float *camera_height) {
+  float base_h = default_camera_height;
+  float camera_offset = 0.0f;
+
+  apollo::perception::onboard::FusionCameraDetection
+  fusion_camera_detection_param;
+  if (!GetProtoConfig(&fusion_camera_detection_param)) {
+    AINFO << "load fusion camera detection component proto param failed";
+    return false;
+  }
+  std::string lidar_type_name_str =
+      fusion_camera_detection_param.lidar_type_name();
```

Comment:
For readability: what about just name "lidar_type_name_str" and "lidar_type_name" to  "lidar_type" ?